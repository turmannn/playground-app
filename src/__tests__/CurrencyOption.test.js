import React from 'react'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
import CurrencyOption from '../components/currencyOption'
import constants from '../data/constants'

configure({adapter: new Adapter()});

describe('CurrencyOption', () => {
  it('should render currency option', () => {
    // when
    const component = shallow(
      <CurrencyOption
        label='some label'
        value='100'
        options={['opt1', 'opt2']}
      />
      )

    //then
    expect(component.getElements()).toMatchSnapshot();
  })
})


//<label htmlFor={props.label.split(' ').join('-')}>{props.label}:</label>
//       <select onChange={props.onChange} value={props.value}>
//         { props.options.map(opt => <option>{opt}</option>) }
//       </select>