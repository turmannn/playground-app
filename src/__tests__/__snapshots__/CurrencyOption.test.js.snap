// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencyOption should render currency option 1`] = `
Array [
  <div>
    <label
      htmlFor="some-label"
    >
      some label
      :
    </label>
    <select
      value="100"
    >
      <option>
        opt1
      </option>
      <option>
        opt2
      </option>
    </select>
  </div>,
]
`;